cc-cli/Program.cs

```diff
@@ -290,23 +282,35 @@ static void Main(string[] args)
                             {
                                 if (argHandler.ArgData.OutFile != null)
                                 {
+                                    if(File.Exists(argHandler.ArgData.OutFile))
+                                    {
+                                        File.Delete(argHandler.ArgData.OutFile);
```

Comment:
Using System.IO.File is not something I am crazy about because it means that we can't run a unit test against it. I know it is currently implemented as a program executable, however I would like to see us start to migrate it towards a library application driven by some argument handler.
There is nothing to do yet, however I would like to call attention to it for future work.

Reply:
That makes sense.